### PR TITLE
Update export.md

### PR DESCRIPTION
**Pull request by yoannchaudet**

### CODE REVIEW

**Review by blparr — APPROVED** [review]

File: docs/analytics/export.md [hunk 1]
```diff
@@ -183,15 +183,33 @@ The table below shows the field mapping for the "customDimensions" field.
...
 | Application Insights            | App Center                                     |
 | ------------------------------- | ---------------------------------------------- |
-|  MessageType                    | Type of event (session, event, ...)            |
+|  AppBuild                       | Application build number                       |
 |  AppId                          | App Center App ID                              |
-|  CarrierCountry                 | Carrier Country                                |
-|  TimeZoneOffset                 | Time Zone Offset                               |
-|  CarrierName                    | Carrier Type                                   |
+|  AppNamespace                   | Application namespace                          |
+|  CarrierCountry                 | Carrier country                                |
```
Comment (guperrot) [inline docs/analytics/export.md:190]: whats the difference between carrierCountry and countryCode? SDK sends only carrierCountry.

File: docs/analytics/export.md [hunk 2]
```diff
@@ -183,15 +183,33 @@ The table below shows the field mapping for the "customDimensions" field.
...
+|  CarrierName                    | Carrier type                                   |
+|  CountryCode                    | Two-letter country code                        |
+|  EventId                        | App Center Event ID                            |
+|  IngressTimestamp               | Log ingestion timestamp                        |
+|  IsTestMessage                  | Flag indicating if the message is a test one   |
```
Comment (guperrot) [inline docs/analytics/export.md]: sounds like internal flag, will always be false for customers, do we want to expose that?

File: docs/analytics/export.md [hunk 3]
```diff
@@ -183,15 +183,33 @@ The table below shows the field mapping for the "customDimensions" field.
...
+|  LiveUpdateDeploymentKey        | App Center Code Push deployment key            |
+|  LiveUpdatePackageHash          | App Center Code Push package hash              |
+|  LiveUpdateReleaseLabel         | App Center Code Push release label             |
```
Comment (guperrot) [inline docs/analytics/export.md]: We might want to remove the liveupdate fields from export, I'm not sure they were ever set.

File: docs/analytics/export.md [hunk 4]
```diff
@@ -183,15 +183,33 @@ The table below shows the field mapping for the "customDimensions" field.
...
+|  Locale                         | Device language                                |
+|  MessageId                      | App Center message identifier                  |
```
Comment (guperrot) [inline docs/analytics/export.md]: What is that? It's not a sdk field.

File: docs/analytics/export.md (hunk 1 above)
Comment (yoannchaudet) [reply docs/analytics/export.md:190]: Must be internal mapping. Removing CountryCode.

File: docs/analytics/export.md (hunk 2 above)
Comment (yoannchaudet) [reply docs/analytics/export.md]: Fair enough. Removing.

File: docs/analytics/export.md (hunk 3 above)
Comment (yoannchaudet) [reply docs/analytics/export.md]: 👍

File: docs/analytics/export.md (hunk 4 above)
Comment (yoannchaudet) [reply docs/analytics/export.md]: Must be an internal field. Removing too.

**Review by jwargo — APPROVED** [review]
Always change the file date with every publish. Changed the link text as "this API" isn't enough information for sight-impared readers.

**Comment by jwargo** [comment]
@yochaude ready to merge?

**Pull request merged**